src/components/header/Header.vue

```diff
@@ -49,6 +52,8 @@ defineProps<Props>()
 //Add emit to modify data in parent component
 const emit = defineEmits(['addTodo', 'onFilterTodos', 'sortTodos', 'reverseTodos'])
 
+const whichArrow = ref('up')
```

Comment:
Can you rename the `whichArrow` variable and the prop of the TodoSport as well to `selectedArrow` or something like this? The `whichArrow` sounds a bit weird for me.